benchmarks/01_trivial/04_fastdiv.ispc

```diff
@@ -0,0 +1,121 @@
+
+
+export void fastdiv_int32_positive_13(uniform int32 *uniform dst, uniform int count) {
```

Comment:
I suggest having single macro for generating thsse functions.

Reply:
done